pkg/operator/controller/status.go

```diff
@@ -108,7 +110,8 @@ func (r *reconciler) getOperatorState() (*corev1.Namespace, []operatorv1.Ingress
 }
 
 // computeOperatorStatusConditions computes the operator's current state.
-func computeOperatorStatusConditions(conditions []configv1.ClusterOperatorStatusCondition, ns *corev1.Namespace, ingresses []operatorv1.IngressController) []configv1.ClusterOperatorStatusCondition {
+func computeOperatorStatusConditions(conditions []configv1.ClusterOperatorStatusCondition, ns *corev1.Namespace,
+	ingresses []operatorv1.IngressController, versionsEqual bool, version string) []configv1.ClusterOperatorStatusCondition {
```

Comment:
What do you think about making the old and new version inputs to `computeOperatorStatusConditions` and letting it do the comparison?

Reply:
Done